tox.ini

```diff
@@ -1,12 +1,8 @@
 [tox]
 envlist =
-    py{36,37,38,39,310}-test{,-conda}
-    build_docs
-    pycodestyle
-requires =
-    setuptools >= 30.3.0
-    pip >= 19.3.1
-isolated_build = true
+    py{38,39,310,311,312}-test{,-conda}{,-devdeps}{,-osxclang,-linuxgcc}
```

Comment:
I think the order is flipped?
```suggestion
    py{38,39,310,311,312}-test{,-osxclang,-linuxgcc}{,-conda}{,-devdeps}
```

Reply:
Order doesn't matter for tox factors?

Reply:
I dunno actually but it is easier to read if in the same order as how it is actually used. Thanks!